local/kind/setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The pipelines-service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+prechecks () {
+  if ! command -v kind &> /dev/null
+  then
+    printf "Kind could not be found\n"
+    exit
+  fi
+}
+
+mk_tmpdir () {
+  TMP_DIR="$(mktemp -d -t kind-pipelines-service.XXXXXXXXX)"
+  printf "Temporary directory created: ${TMP_DIR}\n"
+}
+
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+pushd "$parent_path"
+
+prechecks
+
+printf "Preparing kind clusters\n"
+
+mk_tmpdir
+
+CLUSTERS=(
+    us-east1
+    us-west1
+)
+
+EXISTING_CLUSTERS=$(sudo kind get clusters 2>/dev/null)
```

Comment:
@psturc this would be possible with rootless podman. There are however challenges with the [pid controller](https://bugzilla.redhat.com/show_bug.cgi?id=1897579) besides the fact that cgroup v2 needs to be enabled, which is the default with the latest Fedora versions.